tests/unittests/BackendTest.cpp

```diff
@@ -207,9 +207,9 @@ TEST(Context, basicContextTest) {
 
   // Create a simple graph, just to have a few placeholders.
   Function *F = mod.createFunction("main");
-  auto *input1 = mod.createPlaceholder(ty, "input1");
-  auto *input2 = mod.createPlaceholder(ty, "input2");
-  auto *input3 = mod.createPlaceholder(ty, "input3");
+  auto *input1 = mod.createPlaceholder(ty, "input1", false);
```

Comment:
Should we default isTrainable to false?

Reply:
I considered 3 alternatives here. 1) copy the Variable interface which used a default parameter (the default was True). I was unhappy with the default and did not want to switch the default during the migration. 2) I considered using a different API: createTrainablePlaceholder. and 3) I considered using an enum. I decided to be conservative here because I'll need to update hundreds of unit tests soon, so I prefer to keep it simple an clean it up in stages.

Reply:
Sounds good.